test/conversion_helper-test.js

```diff
@@ -74,6 +74,27 @@ describe('conversion_helper', () => {
           ch.preprocessComment(comment)['body'].should.equal("post text 307.68 inches");
         });
     });
+    context('comment contains multiplier', () => {
```

Comment:
Can we make a new test, to make sure we are matching case-insensitively? "Something like 40K miles"
Also a test for if the input has commas "Something like 4,000k miles"
Ohh, also a test for multiple at the same time!! "Something like 4k miles 5m miles"

Reply:
Regarding commas, shouldn't we also preprocess numbers with commas and replace them with dots (`4,32 -> 4.32`)? Since using commas is prone to cause errors, for example parseFloat('4,32') = 4

Reply:
Now that i think about it they're also used as thousand separators, I don't know what should we do about it.

Reply:
Hmm, you're right, getting rid of commas might make sense in a preprocess step. You can skip the "4,000k miles" test if you would like

Opening separate issue #74 to discuss preprocessing commas, I can see a few tricky bits